examples/02-Advanced-workflow.ipynb

```diff
@@ -0,0 +1,1060 @@
+{
```

Comment:
<p>We should explain more that Lambda Op is an operator to define custom functions. It works only on row-wise operations</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/NVTabular/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
a few words of description added